### DISCUSSION TITLE
Exception logging

### DISCUSSION THREAD

**Discussion by amitmi704** [nornir-automation/nornir#782] [Ideas]
Would it be possible to add the ability to mute exception printing to stdio, or remove it all together?  The storage of the exception into the result is sufficient.

**Comment by ktbyers** [comment]
Turn on Nornir logging and provide a log file, I think that will cause it to stop being written to stdout. I believe this is an aspect of Python logging i.e. if there is no logging destination it writes it to standard out.

Let me know if that works.
